manual/manual.adoc

```diff
@@ -433,7 +438,7 @@ image:../sprites/arm1a0.png[Force Field Armor Vest] |
 image:../sprites/arm2a0.png[Attuned Force Field Armor Vest]
 |==========================
 
-Normal armor absorbs one third of damage you receive. Absorption is rounded down:
+Normal armor absorbs 1/3 of damage you receive. Absorption is rounded down:
```

Comment:
That said, it is inconsistent and most things seem to use words so I'll see if this can be changed without leaving a one-word line at the end of the paragraph (which was the main motivation for this in the first place).